gengo/gengo.py

```diff
@@ -415,16 +415,27 @@ def _raiseForMultipleErrorResponse(self, results):
         error_code = error_codes[0] if error_codes else None
         raise GengoError(' '.join(messages), error_code)
 
-    def _raiseForSingleErrorResponse(self, results):
-        raise GengoError(results['err']['msg'], results['err']['code'])
+    def _raiseForSingleErrorResponse(self, results, code):
```

Comment:
`code` is a little bit vague. maybe something like `status_code` or `http_code`? the target is at first glance, we know the "code" came from the actual response. :thinking:

Reply:
Yeah, I think so, will just change it to `http_code`, for better understanding.